libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -297,7 +299,7 @@ bool ControlFlowBuilder::visit(FunctionCall const& _functionCall)
 				_functionCall.expression().accept(*this);
 				ASTNode::listAccept(_functionCall.arguments(), *this);
 
-				m_currentNode->functionCalls.emplace_back(&_functionCall);
+				m_currentNode->functionCall = &_functionCall;
```

Comment:
```suggestion
				yulAssert(!m_currentNode->functionCall, "");``
				m_currentNode->functionCall = &_functionCall;
```
maybe :-)? Then I don't need to check if it's really only ever at most one :-).

Reply:
Maybe not `yul...` but yeah, I'll add an assert

Reply:
Ok, it creates a new node afterwards right below and it's never set anywhere else, so it's fine... but still wouldn't hurt to assert.

Reply:
> Maybe not `yul...` but yeah, I'll add an assert

Habit :-).